### DISCUSSION TITLE
Deploying to remote iOS device

### DISCUSSION THREAD

**Discussion by lylerolleman** [unoplatform/uno#13332] [General]
When trying to deploy to a remote iOS device connected to a mac from VS on my Windows PC, I am getting an error while building

Fatal error in IL Linker
2>Unhandled exception. System.IO.FileNotFoundException: Could not find file '/Users/XXXX/Library/Caches/Xamarin/mtbs/builds/UnoTestApp.Mobile/7c3351bc3a85080cdffe2170c937a04b6507b80fba62eb886ff81bd56aefff7d/C:/Program Files/dotnet/packs/Microsoft.NETCore.App.Runtime.Mono.ios-arm64/7.0.10/runtimes/ios-arm64/lib/net7.0/Microsoft.Win32.Registry.dll'.

This seems really weird, with the absolute windows path in the unix path on the mac. 

Has anyone observed this and knows how to deal with it?

**Comment by jeromelaban** [comment]
Thanks for the report. This is likely a .NET SDK issue, but which version of .NET are you using?

**Reply by lylerolleman** [reply]
.NET 7

I created the projects using the "dotnet new unoapp -o UnoTestApp" command

I have made no changes from the project structure it produced

**Reply by jeromelaban** [reply]
Thanks. Which version of visual studio are you using?

**Reply by lylerolleman** [reply]
Visual Studio 2022 17.7.1

**Reply by jeromelaban** [reply]
Thanks. This is indeed a very curious issue. Could you [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html) of your build?
